src/arch/be_m1/CMakeLists.txt

```diff
@@ -19,14 +19,11 @@ if("${FORTE_ARCHITECTURE}" STREQUAL "Bachmann M1")
 
 
   forte_add_include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-  if(FORTE_FAKE_TIME)
-    forte_set_timer(../fake_time/faketimerha)
-  else(FORTE_FAKE_TIME)
-    forte_set_timer(../posix/pctimeha)
-  endif(FORTE_FAKE_TIME)
+  
+  forte_add_timerhandler(CPCTimerHandler pctimeha )
```

Comment:
how is the factory known that the `pctimeha` file is in the `../posix/` directory?

Reply:
mmm it seems to be a bad copy + paste